app/kafka/manager/KafkaManager.scala

```diff
@@ -188,10 +188,12 @@ class KafkaManager(akkaConfig: Config)
   }
 
   //--------------------Commands--------------------------
-  def addCluster(clusterName: String, 
+  def addCluster(clusterName: String,
                  version: String, 
                  zkHosts: String, 
                  jmxEnabled: Boolean,
+                 jmxUser: Option[String] = Some(""),
```

Comment:
don't need default value here since this is always passed in (both user and pass).